options/locale/locale_nl-NL.ini

```diff
@@ -1048,7 +1048,7 @@ issues.action_assignee=Toegewezene
 issues.action_assignee_no_select=Geen verantwoordelijke
 issues.opened_by=%[1]s geopend door <a href="%[2]s">%[3]s</a>
 issues.closed_by=door <a href="%[2]s">%[3]s</a> gesloten %[1]s
-issues.closed_by_fake=met %[2]gesloten %[1]s
+issues.closed_by_fake=met %[2]s gesloten %[1]s
```

Comment:
Now I'm completely weirded out, I cannot even find the Netherlands locale in Crowdin…

Reply:
https://crowdin.com/translate/gitea/68/enus-nl